linux_fixes.md

```diff
@@ -0,0 +1,96 @@
+# Linux Tips
```

Comment:
This doc should be split into a few different ones, all contained within a directory called `linux` and linked to independently from the README.

- Common issues
- Battery Tuning
- Advanced Battery Tuning (put the stuff about the TKG kernel here)